vis/assets-src/stylesheets/design-patterns/_search-form.scss

```diff
@@ -28,10 +29,32 @@
     display: block;
     margin-bottom: 15px;
     width: 100%;
-
+    background: url('../images/homepage-background.png');
+    background-size: contain;
+    background-repeat: no-repeat;
+    height: 0;
+    padding-top: 30.34%; /* (img-height / img-width * container-width) */
+    /* (853 / 1280 * 100) */
+    overflow: hidden;
+    
+    @include device-pixel-ratio() {
+      background-image: url('../images/homepage-background@2x.png');
+      background-size: 100% auto;
+    }
+    
     @include media(desktop) {
       border-bottom: 0;
       margin-bottom: 0;
+      background-size: cover;
+      -webkit-background-size: cover;
+      background-position: right;
+      height: 366px;
+      padding-top: 0;
+      
+      @include ie-lte(8) {
+        color: red !important;
```

Comment:
Feels like this just got forgotten to be removed?

Reply:
Ah, yes left over from testing. Now removed!